transcribe.py

```diff
@@ -131,9 +132,11 @@ def main(args, ext_json = ['.json', '.json.gz']):
 		try:
 			tic = time.time()
 			y, ylen = y.to(args.device), ylen.to(args.device)
+			print('Input path:', meta[0]['audio_path'])
```

Comment:
please move these prints before the `y` moves to device to guard against bad OOM

Reply:
done